### PR TITLE
chore(workflow-beta3): releasing workflow-beta3

### PR DESCRIPTION
**Pull request by bacongobbler**

### CODE REVIEW

File: workflow-beta3-e2e/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Workflow 2.0.0-beta2 e2e Tests (parallel)
```
Comment (mboersma) [inline workflow-beta3-e2e/README.md]: This should say "Workflow 2.0.0-beta**3** e2e Tests (parallel)".

File: workflow-beta3/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+name: workflow-beta2
```
Comment (mboersma) [inline workflow-beta3/Chart.yaml]: `s/beta2/beta3/g` here and elsewhere.

File: workflow-beta3/README.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Workflow 2.0.0-beta2
```
Comment (mboersma) [inline workflow-beta3/README.md]: beta3

File: workflow-beta3-e2e/tpl/workflow-e2e-pod.yaml [hunk 4]
```diff
@@ -0,0 +1,32 @@
+#helm:generate helm template -o $HELM_GENERATE_DIR/manifests/workflow-e2e-pod.yaml -d $HELM_GENERATE_DIR/tpl/generate_params.toml $HELM_GENERATE_DIR/tpl/workflow-e2e-pod.yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: workflow-beta2-e2e
```
Comment (mboersma) [inline workflow-beta3-e2e/tpl/workflow-e2e-pod.yaml]: beta3

File: workflow-beta3/Chart.yaml [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+home: https://github.com/deis/workflow
+version: 2.0.0-beta2
```
Comment (mboersma) [inline workflow-beta3/Chart.yaml]: beta3

File: workflow-beta3/tpl/deis-builder-rc.yaml [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+              value: "quay.io/{{.dockerbuilder.org}}/dockerbuilder:{{.dockerbuilder.dockerTag}}"
+            - name: "SLUG_BUILDER_IMAGE_PULL_POLICY"
+              value: "{{.slugbuilder.pullPolicy}}"
+            - name: "DOCKER_BUILDER_IMAGE_PULL_POLICY"
+              value: "{{.dockerbuilder.pullPolicy}}"
+            # This var needs to be passed so that the minio client (https://github.com/minio/mc) will work in Alpine linux
+            - name: "DOCKERIMAGE"
+              value: "1"
+            - name: "DEBUG"
+              value: "false"
```
Comment (mboersma) [inline workflow-beta3/tpl/deis-builder-rc.yaml:43]: THANK YOU. 😀

Comment (bacongobbler) [inline workflow-beta3/tpl/deis-builder-rc.yaml:43]: We should add this to the release checklist.

Comment (arschles) [inline workflow-beta3/tpl/deis-builder-rc.yaml:43]: https://github.com/deis/workflow/issues/197

File: workflow-beta3-e2e/tpl/workflow-e2e-pod.yaml (hunk 4 above)
Comment (mboersma) [inline workflow-beta3-e2e/tpl/workflow-e2e-pod.yaml]: `name: workflow-beta3-e2e`

**Comment by bacongobbler** [comment]
pinging for one last review. Manual testing is complete and the images have been pushed to quay.

File: workflow-beta3/Chart.yaml [hunk 7]
```diff
@@ -0,0 +1,11 @@
+name: workflow-beta3
+home: https://github.com/deis/workflow
+version: 2.0.0-beta3
+description: For testing only!
+maintainers:
+- Deis Team <engineering@deis.com>
+details: |-
+  WARNING: this chart is for testing only! Features may not work, and there are likely to be bugs.
```
Comment (mboersma) [inline workflow-beta3/Chart.yaml:8]: Should we note that workflow-beta3 requires [`helm` 0.6.0](https://github.com/helm/helm/releases/tag/0.6.0) or later?

File: workflow-beta3/README.md [hunk 8]
```diff
@@ -0,0 +1,14 @@
+# Workflow 2.0.0-beta3
+
+WARNING: this chart is for testing only! Features may not work, and there are likely to be bugs.
```
Comment (mboersma) [inline workflow-beta3/README.md:3]: Actually this would be a better place to note the helm requirement. Or both.

Comment (bacongobbler) [inline workflow-beta3/README.md:3]: both.

**Comment by bacongobbler** [comment]
done. Both documents now note that Helm v0.6.0+ is required to use this chart. :)

File: workflow-beta3-e2e/Chart.yaml [hunk 9]
```diff
@@ -0,0 +1,13 @@
+name: workflow-dev-e2e
```
Comment (mboersma) [inline workflow-beta3-e2e/Chart.yaml]: Should be `workflow-beta3-e2e` right?

**Comment by mboersma** [comment]
Re-tested, as well the e2e chart. :ship:

**Pull request merged**